src/test/resources/com/puppycrawl/tools/checkstyle/checks/metrics/InputNPathComplexity.java

```diff
@@ -0,0 +1,162 @@
+package com.puppycrawl.tools.checkstyle.checks.metrics;
+// Advise: for lack of ambiguity try to make all factors prime numbers
+public class InputNPathComplexity {
+    //NP = 1 + 4 = 5
+    void testIfWithExpression() {
+        //1 (default way + 4 (for conditionals)
+        if (true && true || (true || true)) { }
```

Comment:
this Check is very complicated so lets follow formulas that we defined in our 
http://checkstyle.sourceforge.net/config_metrics.html#NPathComplexity

> NP(if-range) + NP(expr) + 1
P(i=1:i=N)NP(Statement(i))

so comment over IF should be `(if-range=1) + 1 + (expr=4) = 6` . So 6 is npath value for IF only.
NPath value for method is 6 so comment should be `NP = 6`
Please keep this notation to clearly define to what part of code certain value is corresponding.